tests/checker/good/gold/adt_test.scilla.gold

```diff
@@ -6,7 +6,59 @@
     "params": [],
     "fields": [],
     "transitions": [],
-    "events": []
+    "events": [],
+    "ADTs": [
+      {
+        "tname": "TestType1",
+        "tparams": [],
+        "tmap": [
+          { "cname": "Item1", "argtypes": [ "Uint32" ] },
+          { "cname": "Item2", "argtypes": [] }
+        ]
+      },
+      {
+        "tname": "Option",
```

Comment:
Should we output the built-in ADTs?

I can see the point of doing it, and it might make life simpler for the IDE developer, but it also clutters up the output. Maybe @iantanwx has an opinion on this?

Reply:
All of this contract info kind of output we have is useful almost only for the IDE. So clutter is okay.

On the other hand, for others using the checker on the command line, we can provide a flag to switch on contract info, and keep it off by default. The IDE will always provide the flag.

If that seems like a good idea, I'll create an Issue for it (it should be simple, but I don't want to mix the two in the same PR).

Reply:
That make sense.

Reply:
Created #448